pkg/app/pipectl/cmd/event/register.go

```diff
@@ -79,3 +86,13 @@ func (r *register) run(ctx context.Context, input cli.Input) error {
 	)
 	return nil
 }
+
+func (r *register) validateEventContexts() error {
```

Comment:
This function looks testable, please add tests for this later (another PR is ok, since this one is for rc release)

Reply:
cc @ffjlabo